src/applications/static-pages/cta-widget/components/messages/MFA.jsx

```diff
@@ -1,29 +0,0 @@
-import React from 'react';
```

Comment:
This component is no longer needed as their is a new CTA that is being used via: `'./components/messages/DirectDeposit/MFA'`